resources/serverless/templates/function-dashboard.yaml

```diff
@@ -594,8 +606,9 @@ data:
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(istio_requests_total{destination_service_namespace=\"$environment\", destination_service_name=\"$lambda_service\"}[2m])) by (destination_service, response_code)",
-              "legendFormat": "{{ `{{response_code}}` }}",
+              "expr": "sum(rate(istio_requests_total{destination_service_namespace=\"$namespace\", destination_service_name=\"$function_revision\"}[2m])) by (destination_service, response_code)",
+              "interval": "",
+              "legendFormat": "{{"{{"}}response_code}}",
```

Comment:
That change looks wrong, I guess it should stay as it was.
The escaping is required for helm templating, after templating this needs to be there: `{{response_code}}`